### PR TITLE
Dockerize cloud app

### PR DESCRIPTION
**Pull request by smahmed776**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "flojoy-cloud",
   "scripts": {
-    "build": "turbo run build",
+    "fe:build": "turbo run build --filter={./apps/frontend}",
```
Comment (itsjoeoui) [inline package.json:4]: what is the reason for filter here?

**Review by itsjoeoui — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (smahmed776) [reply package.json:4]: @itsjoeoui To specify workspace location where to run build command from. Are you able to build frontend from root without specifying this option?

Comment (itsjoeoui) [reply package.json:4]: Yep it works for me without any tweaks

Comment (smahmed776) [reply package.json]: It worked for me too now :) Thanks. I'm reverting...

**Review by itsjoeoui — APPROVED** [review]
lgtm

**Pull request merged**